packages/expo-updates/android/src/main/java/expo/modules/updates/UpdatesModule.kt

```diff
@@ -20,7 +20,12 @@ import expo.modules.updates.manifest.UpdateManifest
 
 // these unused imports must stay because of versioning
 /* ktlint-disable no-unused-imports */
-import expo.modules.updates.UpdatesConfiguration
```

Comment:
this line needs to stay (as indicated by the comments) even though the IDE might automatically remove it

Reply:
Fixed